src/commands.ts

```diff
@@ -467,6 +467,35 @@ export function registerCommands(
 		}),
 	);
 
+	context.subscriptions.push(
+		vscode.commands.registerCommand('pr.pickOnVscodeDev', async (pr: PRNode | DescriptionNode | PullRequestModel) => {
+			if (pr === undefined) {
+				// This is unexpected, but has happened a few times.
+				Logger.error('Unexpectedly received undefined when picking a PR.');
+				return vscode.window.showErrorMessage(vscode.l10n.t('No pull request was selected to checkout, please try again.'));
+			}
+
+			let pullRequestModel: PullRequestModel;
+
+			if (pr instanceof PRNode || pr instanceof DescriptionNode) {
+				pullRequestModel = pr.pullRequestModel;
+			} else {
+				pullRequestModel = pr;
+			}
+
+			const fromDescriptionPage = pr instanceof PullRequestModel;
+			/* __GDPR__
+			"pr.checkout" : {
+				"fromDescriptionPage" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
+				"location" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
+			}
+		*/
+			telemetry.sendTelemetryEvent('pr.checkout', { fromDescription: fromDescriptionPage.toString(), location: 'vscode.dev' });
+
```

Comment:
Let's not include the telemetry event here and keep it just for when a PR is checked out locally.

Reply:
Should we still have some telemetry event for this?

Reply:
We basically already do since we have telemetry for when commands are run.